helm/charts/hpe-csi-driver/templates/sc.yaml

```diff
@@ -1,11 +1,16 @@
+{{ if .Values.storageClass.create -}}
 ---
 apiVersion: storage.k8s.io/v1
 kind: StorageClass
 metadata:
- name: hpe-csi-sc-test
- namespace: default
- labels:
-   plugin: {{ .Release.Name }}
+   name: {{ .Values.storageClass.name }}
+   namespace: default
```

Comment:
Storage classes are not namespaced. Please omit.

Reply:
done..